testing/trino-product-tests/src/main/java/io/trino/tests/product/hudi/TestHudiSparkCompatibility.java

```diff
@@ -240,6 +275,57 @@ public void testMergeOnReadTableSelectAfterUpdate()
         }
     }
 
+    @Test(groups = {HUDI, PROFILE_SPECIFIC_TESTS})
+    public void testMergeOnReadTableContainsReplaceCommit()
+    {
+        String tableName = "test_hudi_mor_contains_replace_commit_" + randomNameSuffix();
+
+        createNonPartitionedTable(tableName, MOR_TABLE_TYPE);
+
+        List<QueryAssert.Row> expectedRows = ImmutableList.of(
+                row(1, "a1", 20, 1000),
+                row(2, "a2", 40, 2000));
+
+        List<QueryAssert.Row> expectedRows2 = ImmutableList.of(
+                row(1, "a1", 20, 1000),
+                row(2, "a2", 40, 2000),
+                row(3, "a3", 60, 3000),
+                row(4, "a4", 80, 4000));
+
+        try {
+            onHudi().executeQuery(format(
+                    """
+                            ALTER TABLE default.%s
+                            SET SERDEPROPERTIES (
+                                hoodie.parquet.small.file.limit=0,
+                                hoodie.compact.inline=true,
+                                hoodie.compact.inline.max.delta.commits=2,
+                                hoodie.clustering.inline=true,
+                                hoodie.clustering.inline.max.commits=2)""",
+                    tableName));
+            assertThat(onHudi().executeQuery("show TBLPROPERTIES default." + tableName))
```

Comment:
Same here, validating the properties is not necessary.

Reply:
Removed it